class DoesNotExistError(Exception):
    pass


class AlreadyExistsError(Exception):
    pass
